org.ethereum.Mist.json

```diff
@@ -0,0 +1,64 @@
+{
+    "app-id": "org.ethereum.Mist",
+    "branch": "stable",
+    "base": "io.atom.electron.BaseApp",
+    "base-version": "stable",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "mist",
+    "appstream-compose": true,
+    "separate-locales": false,
+    "finish-args": [
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--talk-name=org.freedesktop.Notifications",
+        "--share=ipc",
+        "--share=network",
+        "--filesystem=xdg-download"
+    ],
+    "modules": [
+        {
+            "name": "mist",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -D org.ethereum.Mist.desktop /app/share/applications/org.ethereum.Mist.desktop",
+                "install -D org.ethereum.Mist.appdata.xml /app/share/appdata/org.ethereum.Mist.appdata.xml",
+                "install -D icon.png /app/share/icons/hicolor/256x256/apps/org.ethereum.Mist.png",
+                "install -D mist /app/bin/mist",
+
+                "unzip Mist-*.zip",
+                "install -d /app/share/Mist",
+                "cp -a linux-unpacked/* /app/share/Mist"
+            ],
+            "sources": [
+                {
+                    "type": "script",
+                    "dest-filename": "mist",
+                    "commands": [
+                        "exec env TMPDIR=$XDG_CACHE_HOME /app/share/Mist/mist $@"
+                    ]
+                },
+                {
+                    "type": "file",
+                    "url": "https://github.com/ethereum/mist/releases/download/v0.9.3/Mist-linux64-0-9-3.zip",
```

Comment:
Create a `flathub.json` file containing:

```json
{
  "only-arches": ["x86_64"]
}

Reply:
Is there a way to have the manifest download the 32 bit version when building for that platform?

Reply:
Yes there is an `only-arches` property for sources. See `man flatpak-manifest`

Reply:
You will still want that file because they probably don't ship arm binaries. Just add `"i386"` to the list.